Docs/deployment/configuration.md

```diff
@@ -0,0 +1,118 @@
+# Configuration
+
+Two configuration methods can be used to control Santa: local configuration and a sync server controlled configuration.
+
+#### Local Configuration
+
+| Key                           | Value Type | Description                              |
+| ----------------------------- | ---------- | ---------------------------------------- |
+| ClientMode*                   | Integer    | 1 = MONITOR, 2 = LOCKDOWN, defaults to MONITOR |
+| FileChangesRegex*             | String     | The regex of paths to log file changes. Regexes are specified in ICU format. |
+| WhitelistRegex*               | String     | A regex to whitelist if the binary or certificate scopes did not allow execution.  Regexes are specified in ICU format. |
+| BlacklistRegex*               | String     | A regex to blacklist if the binary or certificate scopes did not block an execution.  Regexes are specified in ICU format. |
+| EnablePageZeroProtection      | Bool       | Enable `__PAGEZERO` protection, defaults to YES.  If this flag is set to NO, 32-bit binaries that are missing  the `__PAGEZERO` segment will not be blocked. |
```

Comment:
Reverse the description and mention that this can be over-riden by rules.

> If this flag is set to YES, 32-bit binaries that are missing the `__PAGEZERO` segment will be blocked even in MONITOR mode, **unless** the binary is whitelisted by an explicit rule.